src/black/__init__.py

```diff
@@ -112,9 +112,7 @@ def read_pyproject_toml(
     try:
         config = parse_pyproject_toml(value)
     except (OSError, ValueError) as e:
-        raise click.FileError(
-            filename=value, hint=f"Error reading configuration file: {e}"
-        )
+        raise click.UsageError(f"unable to read configuration file ({value}): {e}")
```

Comment:
What is the reason for this change?

Reply:
Sorry, it's in the PR description.

Reply:
During manual testing I didn't really like the existing error message when an error happens during reading or parsing:

main:

```console
❯ black --config=tests/invalid.toml
Error: Could not open file 'tests/invalid.toml': Error reading configuration file: Expected "=" after a key in a key-to-value mapping
```

this PR:

```console
❯ black --config=tests/invalid.toml
Usage: black [OPTIONS] [SRC]...
Try 'black -h' for help.

Error: unable to read configuration file (tests/invalid.toml): Expected "=" after a key in a key-to-value mapping
```

Reply:
I actually prefer the message from main. The usage information is noise here, since the problem isn't with the command line flag syntax, but with the contents of the config file.

Reply:
Fair enough, I'll revert.